packages/commercetools/composables/src/useForgotPassword/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Context, UseForgotPassword, useForgotPasswordFactory, UseForgotPasswordFactoryParams } from '@vue-storefront/core';
+
+const useForgotPasswordFactoryParams: UseForgotPasswordFactoryParams<any> = {
+  resetPassword: async (context: Context, { email, customQuery }) => {
+    try {
+      return await context.$ct.api.customerCreatePasswordResetToken(email, customQuery);
+    } catch (err) {
+      err.message = err?.graphQLErrors?.[0]?.message || err.message;
```

Comment:
Let's externalize this to simple inline function, so if one day path to graphqlerror changes - you will need to update it only in one place.

Reply:
This is a good idea, however it is not in the scope of current task. I can create a separate issue out of that and we can think about implementing it later.